app/assets/scss/_brief_submission.scss

```diff
@@ -1,3 +1,20 @@
+.instruction-list-item {
```

Comment:
Having different instruction list styling in different apps seems completely the opposite point of having a frontend toolkit?